framework/console/Controller.php

```diff
@@ -222,8 +222,7 @@ public function bindActionParams($action, $params)
                 }
                 $args[] = $actionParams[$key] = $params[$key];
                 unset($params[$key]);
-            } elseif (
-                PHP_VERSION_ID >= 70100
+            } elseif (PHP_VERSION_ID >= 70100
```

Comment:
It was better before the change. Is it part of PSR-12?

Reply:
No, PSR2 only, I did not push anything for PSR12 yet.

Reply:
I think it checks how the first condition is, see few lines before.

Reply:
With PSR12 this line is not touched anymore.